### PR TITLE
cap the urllib3 version to workaround a known issue at requests

### PR DESCRIPTION
**Pull request by yugangw-msft**
@tjprescott, if `https://github.com/requests/requests/issues/4830` stays unfixed before you snap the code to build the windows installer, please merge this, open a tracking issue so to pull it out once a new requests package gets released.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/setup.py [hunk 1]
```diff
@@ -100,6 +100,8 @@
     'azure-cli-sql',
     'azure-cli-storage',
     'azure-cli-vm'
```
Comment (tjprescott) [inline src/azure-cli/setup.py]: Missing comma here.

File: src/command_modules/azure-cli-appservice/HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.2.6
```
Comment (tjprescott) [inline src/command_modules/azure-cli-appservice/HISTORY.rst]: Bump version in setup.py as well

File: src/azure-cli/setup.py (hunk 1 above)
Comment (tjprescott) [reply src/azure-cli/setup.py]: This is still failing CI

**Comment by yugangw-msft** [comment]
requests has released v2.20.0, we can close this pr now

**Pull request closed without merging**